package.json

```diff
@@ -69,18 +69,19 @@
     "es6-promise": "^3.0.2",
     "fast-isnumeric": "^1.1.1",
     "font-atlas-sdf": "^1.3.3",
+    "gl-axes3d": "^1.2.6",
```

Comment:
why are we adding a package but making no direct reference to it?

Reply:
@alexcjohnson good point. I wonder if we should remove `gl-shader` dependency as well, since we're not using it by plotly directly.

Reply:
Sounds reasonable to me, but after the mess we had yesterday with `npm ls` I'd like to figure this out with @etpinard 's input.